service/src/chain_spec.rs

```diff
@@ -126,6 +127,10 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 		curated_grandpa: Some(CuratedGrandpaConfig {
 			shuffle_period: 1024,
 		}),
+		im_online: Some(ImOnlineConfig{
+			gossip_at: 0,
+			last_new_era_start: 0,
```

Comment:
We can remove those two properties from the config in the next version of the module.